buildbot/compile.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+PROJECT=
+BRANCH=
+BUILD_NUMBER=
+PR_NUMBER=
+
+# $1 exit code
+# $2 error message
+exit_if_err()
+{
+    if [ $1 -ne 0  ]; then
+        echo "Error: $2"
+        exit $1
+    fi
+}
+
+unset OPTIND
+while getopts ":p:b:r:n:" option; do
+    case $option in
+        p) PROJECT=$OPTARG ;;
+        b) BRANCH=$OPTARG ;;
+        n) BUILD_NUMBER=$OPTARG ;;
+        r) PR_NUMBER=$OPTARG ;;
+    esac
+done && shift $(($OPTIND - 1))
+
+# we're in llvm.obj dir
+BUILD_DIR=${PWD}
+
+if [ "$PROJECT" == "sycl" ]; then
+    make -j`nproc`
+elif [ "$PROJECT" == "llorg" ]; then
+    make -j`nproc`
+fi
```

Comment:
What is the point of this `if`?
What is the purpose of PROJECT variable?

Reply:
OK, we can remove it.